resources/plane_reserve/intent2reply.json

```diff
@@ -1,4 +1,10 @@
 {
     "hello": "Hello, how can I help?",
     "ask_name": "May I have your name, please?",
+    "plane_ask_flight_id": "Can I have your flight ID, please?",
+    "plane_flight_available": "The flight is available. Should I reserve it for you?",
+    "plane_flight_unavailable": "Sorry, but the flight with id '{flight_id:i}' is not available any more.",
+    "plane_reservation_succeeded": "Right, your reservation completed successfully!",
```

Comment:
This sounds a bit odd. Maybe "Your flight is now reserved." or "Your reservation was successful." Although the first is more specific, and therefore better, I think.